03-task-manager/optionalArrayMethodsReviewExtraAssignment.js

```diff
@@ -184,7 +184,7 @@ console.log('everyone last name', everyonesLastName);
 
 //////// CHALLENGE: Filter to the people who followed the right
 // "right format" is "<first name> <last name>" with a single space!
-const rightFormat = /^\w+ \w+$/;
+const rightFormat = /\w^+ \w+$/;
```

Comment:
Why was this line changed to this?
This regex is invalid.

Reply:
I don't know actually. I noticed a file change but I don't remember doing any line updates in the optional file. I'll change back!